styles/pages/Settings.module.scss

```diff
@@ -0,0 +1,195 @@
+/*
+****************************************
+Settings: Container
+****************************************
+*/
+.settings {
+  margin-top: 34px;
+  padding: 64px;
+  background-color: white;
+}
+
+.settings__title {
+  margin-bottom: 20px;
+}
+
+/*
+****************************************
+Settings: Category selection
+****************************************
+*/
+.settings__categorySelection {
+  // Select category text
+  h2 {
+    font-family: DM Sans;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 26px;
+    margin: 0 0 36px 0;
+
+    color: var(--brand-pop-blue);
+  }
+
+  // Dropdown label
+  h3 {
+    font-family: DM Sans;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 16px;
+    line-height: 21px;
+    margin: 0 0 12px 0;
+
+    letter-spacing: 0.05em;
+    text-transform: uppercase;
+
+    color: var(--system-neutral-grey);
+  }
+
+  // Dropdown select component
+  .settings__categoryDropdown {
+    width: 100%;
+    margin-bottom: 12px;
+  }
+}
+
+// Dropdown category values
+.settings__categoryValues__wrapper {
+  height: 176px;
```

Comment:
For the styling, are we going to be changing to % later?

Reply:
Yeah